### PR TITLE
Auth fixes

### PR DESCRIPTION
**Pull request by indytechcook**
@tladendo @vietnogi @r-jain auth fixes and build fixes.

### CODE REVIEW

File: src/client/app/login/scripts/login.controller.js [hunk 1]
```diff
@@ -15,6 +15,7 @@
     var lock = new Auth0Lock(auth0ClientId, auth0Domain);
     $scope.signin = function () {
       var state = encodeURIComponent('retUrl=' + retUrl);
+      console.log('retUrl', retUrl);
```
Comment (vietnogi) [inline src/client/app/login/scripts/login.controller.js]: remove?

File: src/client/app/login/scripts/login.service.js [hunk 2]
```diff
@@ -5,9 +5,9 @@
     .module('app.login')
     .factory('LoginService', LoginService);
 
-  LoginService.$inject = ['$q', '$http', '$location', 'exception', 'logger', 'apiUrl'];
+  LoginService.$inject = ['$q', '$http', '$location', '$state', '$window', 'exception', 'logger', 'apiUrl'];
   /* @ngInject */
-  function LoginService($q, $http, $location, exception, logger, apiUrl) {
+  function LoginService($q, $http, $location, $state, $window, exception, logger, apiUrl) {
```
Comment (vietnogi) [inline src/client/app/login/scripts/login.service.js]: added $window but no new code to use it?

**Comment by indytechcook** [comment]
Thanks @vietnogi I'v pushed some changes

**Comment by tladendo** [comment]
lgtm

**Pull request merged**